src/access-broker.c

```diff
@@ -522,6 +522,7 @@ access_broker_init_tpm (AccessBroker *broker)
 out:
     return rc;
 }
+
```

Comment:
no superfluous whitespace changes please: https://github.com/tpm2-software/tpm2-abrmd/blob/master/CONTRIBUTING.md#patch-requirements

Reply:
OK, but there is no mention at the link about whitespace, or at the links at the link.
Just trying to make the code more readable.